mail-filter/imapfilter/imapfilter-2.8.2.ebuild

```
@@ -26,7 +26,6 @@ DEPEND="${RDEPEND}"
 DOCS="AUTHORS NEWS README samples/*"
```

please use array notation here. Please also verify the last one works (I don't remember if wildcard works in DOCS, so better verify).

```suggestion
DOCS=( AUTHORS NEWS README "samples/*" )
```

---

It never worked - I tried this in the past when I started maintaining this package, which is why I continued to use the string notation:
```
Traceback (most recent call last):
  File "/usr/lib/portage/python3.12/doins.py", line 609, in <module>
    sys.exit(main(sys.argv[1:]))
             ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.12/doins.py", line 598, in main
    if _doins(
       ^^^^^^^
  File "/usr/lib/portage/python3.12/doins.py", line 441, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.12/doins.py", line 375, in install_file
    return self._ins_runner.run(source, dest_dir)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.12/doins.py", line 183, in run
    sstat = os.stat(source)
            ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: b'samples/*'
```
Not sure if this is intentional, but in any case it won't work without a fix to portage.
For now I will just expand the regex since it's only two files.

---

All suggestions fixed. Thanks!